software/24.Git_GitHub/GitHub/README.md

```diff
@@ -0,0 +1,60 @@
+You have to create 3 issues:
+- One to describe the problem, with expected behaviour and what you are experiencing.
+- Another to suggest a feature (add all the arguments instead of just the first 2)
+- A last one to take a single string as parameter and support operators in it (this one will not be implemented)
+
+Method: result screen to reproduce or fully autonomous with correction after
+
+> Take a look at [labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) and [assignees](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users)
+
+## Step 2 - [Pull Requests (PRs)](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+Congratulations, you solved these issues !  
+But don't close them yet, you will now learn to use Pull Requests and use the option to close issues automatically :sparkles:
+
+Create 2 Pull Requests, and use 2 different methods to link the related issues.
+When it's done, add your teammate as reviewer and play with the review tool in the `Files Changed` tab !
+
+<i>Pro tip:</i>
+
+> Take a look at [Closing keywords](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) 😉
+
+## Step 3 - [Projects](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/about-project-boards)
+You can now manage features and tasks with issues, group commits and do code reviews with PRs, but how do you actually organize
+your work in tables like well-known ticketing tools ? That's where GitHub Projects come into play !
+
+> A [new GitHub Project experience](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/about-projects) is currently in public beta. We chose not to cover it because it's subject to major changes, but it brings great features such as [custom fields](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/about-projects#adding-metadata-to-your-tasks) and [different views](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/about-projects#adding-metadata-to-your-tasks).
+
+## Step 4 - [Milestones](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/about-milestones)
+Organizing taks with states like `To do`, `In progress` and `Done` is great, but how to you handle deadlines and group several issues and PRs
+linked to a sprint for example ? Milestones are exactly what you need !  
+You can find them in the `Issues` tab.
+
+## Step 5 - [Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions)
+<u>Pro tips:</u>
+> If you want to learn more about GitHub Actions, you can check [our dedicated workshop](https://github.com/PoCInnovation/Workshops/tree/24.Git_Github/software/05.Actions) !
+
```

Comment:
You just only quote the documentation, you need to give an objective at your step and show a final result. You need also to explain when we can need to use it.